casper/src/test/scala/coop/rchain/casper/util/rholang/DeployerAuthTest.scala

```diff
@@ -67,14 +70,14 @@ class DeployerAuthTest extends FlatSpec with Matchers {
     ): Assertion = {
       val contract = ConstructDeploy.sourceDeploy(
         source =
-          s"""new auth(`rho:deployer:auth`) in { contract @"checkAuth"(input, ret) = { ret!(*input == *auth) }}""",
+          s"""new auth(`rho:rchain:deployerId`) in { contract @"checkAuth"(input, ret) = { ret!(*input == *auth) }}""",
         timestamp = System.currentTimeMillis(),
         accounting.MAX_VALUE,
         sec = deployer
       )
       val captureChannel = "__RETURN_VALUE__"
       val checkAuth =
-        s"""new auth(`rho:deployer:auth`), ret in {
+        s"""new auth(`rho:rchain:deployerId`), ret in {
```

Comment:
I expected this to happen in `rename deployer auth to deploy id` commit - but no big deal :) I'm bringing this up mainly to make sure you're in control of your rebases :)

Reply:
Yeah, sorry for missing that. But not a rebasing issue :)